src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const regularPrice = 40;
+const amountOfDaysForFiftyOff = 7;
+const amountOfDaysForTwenty = 3;
+
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * regularPrice;
+
+  if (days >= amountOfDaysForFiftyOff) {
+    return price - 50;
```

Comment:
The discount value '50' is hard-coded. Use a constant to define this value, such as 'MAX_DISCOUNT_AMOUNT', for better code maintainability.